git/services/gitRepoInfo.js

```diff
@@ -5,7 +5,7 @@
  * @return {Object} An object containing the github owner and repository name
  */
 module.exports = function gitRepoInfo(packageInfo) {
-  var GITURL_REGEX = /^https?:\/\/[^\/]+\/([^\/]+)\/(.+).git$/;
+  var GITURL_REGEX = /^[git\+h|h]+ttps:\/\/github.com\/([^\/]+)\/(.+).git$/;
```

Comment:
But it is less restrictive at the same time, as it allows `'++ttps://github.com/foo/bar.git'` :stuck_out_tongue:
@perrygovier probably meant `/^(?:git\+https|https?):\/\/[^/]+\/([^/]+)\/(.+).git$/` :smile: